src/main/java/org/dcsa/ovs/model/TransportEvent.java

```diff
@@ -4,43 +4,33 @@
 import com.fasterxml.jackson.annotation.JsonTypeName;
 import lombok.Data;
 import lombok.NoArgsConstructor;
-import org.dcsa.ovs.model.enums.LocationType;
+import org.dcsa.ovs.model.enums.OperationsEventTypeCode;
+import org.dcsa.ovs.model.enums.PortCallServiceTypeCode;
 import org.springframework.data.relational.core.mapping.Column;
 import org.springframework.data.relational.core.mapping.Table;
 
 import java.time.OffsetDateTime;
 import java.util.UUID;
 
-@Table("transport_event")
-@Data
+@Table("operations_event")
```

Comment:
I am a little confused - the file is called TransportEvent.java - but it is using the "operations_event" table?

Reply:
fixed